src/xcode/ENA/ENA/Source/Scenes/Vaccination/VaccinationQRCodeScannerViewModel.swift

```diff
@@ -37,13 +37,22 @@ class VaccinationQRCodeScannerViewModel: NSObject, AVCaptureMetadataOutputObject
 
 		deactivateScanning()
 		guard let code = metadataObjects.first(where: { $0 is MetadataMachineReadableCodeObject }) as? MetadataMachineReadableCodeObject,
-			  let url = code.stringValue,
-			  !url.isEmpty
+			  let fullUrlString = code.stringValue
```

Comment:
I would name it `fullURLString` but that's not a show stopper.

Reply:
it's not even an URL